### PR TITLE
Changes

### LINKED ISSUE #8282: The text size in the player list is massive
Thanks to the text size, the player list can only fit something like five people at the moment. We might wanna scale that down just a tad.

https://i.imgur.com/un9DBVd.png

### PR DESCRIPTION
**Pull request by YunoGasai9862**
### Purpose
It Fixes the text size in the player list, which was previously quite massive.
Fixes #8282
CL: [Improvement] : Text Size

### CODE REVIEW

File: UnityProject/Assets/Materials/Sprites_Outline.mat
Comment (Bod9001) [inline UnityProject/Assets/Materials/Sprites_Outline.mat:1]: You don't need to contribute this stuff

Comment (Bod9001) [reply UnityProject/Assets/Materials/Sprites_Outline.mat:1]: Just need the prefabs you changed

Comment (YunoGasai9862) [reply UnityProject/Assets/Materials/Sprites_Outline.mat:1]: I'm new to this stuff, can you tell me what do you mean? Should i just name the prefabs here?

Comment (MaxIsJoe) [reply UnityProject/Assets/Materials/Sprites_Outline.mat:1]: He means that you're committing unnecessary changes, only commit changes you made intentionally.

**Comment by MaxIsJoe** [comment]
UnityProject/Assets/Materials/DrunkMaterial.mat
UnityProject/Assets/Materials/PostProcess/Post-FloorFovGenerator Material.mat
UnityProject/Assets/Materials/PostProcess/Post-FovGenerator.mat
UnityProject/Assets/Materials/Resources/OverlayCritHole.mat
UnityProject/Assets/Materials/Sprites_Outline.mat
UnityProject/Assets/Prefabs/Items/Clothing/Headwear/Hardhats/HardHat.prefab
UnityProject/Assets/Prefabs/Objects/WallProtrusions/SecurityCamera.prefab
UnityProject/Assets/Prefabs/Player/Resources/Player_V4(uNet).prefab
UnityProject/Assets/Scripts/Core/Sprite Handler/SpriteHandler.cs
UnityProject/Packages/packages-lock.json
UnityProject/ProjectSettings/ProjectSettings.asset

Revert these in git by doing:

1. Open a terminal inside the project that you cloned.
2. Write `git checkout upstream/develop -- path/to/file`
3. replace path/to/file with the paths i've listed above. (Do them one by one)

File: UnityProject/Assets/Materials/Sprites_Outline.mat
Comment (YunoGasai9862) [reply UnityProject/Assets/Materials/Sprites_Outline.mat:1]: I'm sorry. let me do the changes.

**Pull request closed without merging**